From fac20010cbcafe1807a8dce2236727195b3ea868 Mon Sep 17 00:00:00 2001
From: Maxim Kuvyrkov <maxim@codesourcery.com>
Date: Sun, 23 Aug 2009 22:42:33 +0400
Subject: [PATCH] Define libgcc_s.so version for m68k and hppa.

	* sysdeps/m68k/shlib-versions: New file, set libgcc_s' version to 2.
	* sysdeps/hppa/shlib-versions: Add line for libgcc_s, set version to 4.

Signed-off-by: Maxim Kuvyrkov <maxim@codesourcery.com>
---
 sysdeps/hppa/shlib-versions |    2 ++
 sysdeps/m68k/shlib-versions |    1 +
 4 files changed, 11 insertions(+), 0 deletions(-)
 create mode 100644 sysdeps/m68k/shlib-versions

diff --git a/sysdeps/hppa/shlib-versions b/sysdeps/hppa/shlib-versions
index 5a1a865..210b0ab 100644
--- a/ports/sysdeps/hppa/shlib-versions
+++ b/ports/sysdeps/hppa/shlib-versions
@@ -5,3 +5,5 @@ hppa.*-.*-.*		libc=6			GLIBC_2.2
 hppa.*-.*-.*		ld=ld.so.1		GLIBC_2.2
 
 hppa-.*-.*		libBrokenLocale=1	GLIBC_2.2
+
+hppa-.*-.*              libgcc_s=4
diff --git a/sysdeps/m68k/shlib-versions b/sysdeps/m68k/shlib-versions
new file mode 100644
index 0000000..2bb80d8
--- /dev/null
+++ b/ports/sysdeps/m68k/shlib-versions
@@ -0,0 +1 @@
+m68k-.*-.*              libgcc_s=2
-- 
1.6.4

